### PR TITLE
Added changes suggested by goimports

### PR DESCRIPTION
**Pull request by brunats**
**- What I did**
Hi Ritchiers,

In the search for improving the quality of the code I decided to open a PR by passing the `goimports` in the project.

Adds all suggested changes (and revised one by one) to the project.

**- How to verify it**
> Clone this pr with the command
```bash
git fetch upstream pull/509/head:improvement/updates-linters

```
> And run goimports
```bash
git checkout master
goimports -w $(find . -type f -name '*.go' -not -path "./vendor/*") # Check the differences generated
git checkout improvement/updates-linters
goimports -w $(find . -type f -name '*.go' -not -path "./vendor/*") # Check the differences
```

**- Description for the changelog**

Improves code quality related to linter goimport.

### CODE REVIEW

File: pkg/rcontext/remover.go [hunk 1]
```diff
@@ -19,8 +19,9 @@ package rcontext
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/ZupIT/ritchie-cli/pkg/file/fileutil"
 	"strings"
+
+	"github.com/ZupIT/ritchie-cli/pkg/file/fileutil"
 )
```
Comment (victor-schumacher) [inline pkg/rcontext/remover.go:25]: nice

**Review by victor-schumacher — APPROVED** [review]
:1st_place_medal:

**Review by lucasdittrichzup — APPROVED** [review]

**Comment by lucasdittrichzup** [comment]
:rocket:

**Comment by brunasilvazup** [comment]
\merge qa

**Pull request merged**